DOCS-3930 Moving Around the Synthetic Testing Tunnel (#15316)
DataDog/documentation: content/en/synthetics/private_locations/_index.md

You can also **vertically scale** your private locations by increasing the load your private location containers can handle. Similarly, you should use the `concurrency` parameter to adjust the maximum number of test your workers allowed to run and update the resources allocated to your workers.

For more information, see [Dimensioning Private Locations][18]. 

> In order to use private locations for continuous testing, the value you set for the `concurrency` parameter is the concurrency limit. For more information, see [Continuous Testing][23].

Reviewer: ```suggestion
In order to use private locations for continuous testing, the value set in the `concurrency` parameter will control your parallelization. For more information, see [Continuous Testing][23].
```

I will follow up with you on this outside this PR
